extensions/quartz/deployment/src/main/java/io/quarkus/quartz/deployment/QuartzProcessor.java

```diff
@@ -96,7 +97,7 @@ QuartzJDBCDriverDialectBuildItem driver(List<JdbcDataSourceBuildItem> jdbcDataSo
             QuartzBuildTimeConfig config,
             Capabilities capabilities) {
         if (!config.storeType.isDbStore()) {
-            if (config.clustered) {
+            if (config.clustered && !config.storeType.isCustom()) {
```

Comment:
Not all custom store types are going to support clustered jobs, correct?
If so, the idea is to assume that the user know what they are doing? and they'll configure the additional properties accordingly?

Reply:
Yes, Not all custom store types require restrictions to support clustered jobs. 
Job store implementer should know if it is clustered or not, as the JobStore interface only has [JobStore.isClustered()](https://github.com/quartz-scheduler/quartz/blob/master/quartz-core/src/main/java/org/quartz/spi/JobStore.java#L114) getter and no setter method.
Therefore, it can be designed with or without support for clustered property, which are already supported by default build-in implementations of RAM and DB types.

I add comments "When configured with {@link StoreType#CUSTOM}, this property has no effect."

```java
@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
public class QuartzBuildTimeConfig {
    /**
     * Enable cluster mode or not.
     * <p>
     * If enabled make sure to set the appropriate cluster properties.
     * <p>
     * When configured with {@link StoreType#CUSTOM}, this property has no effect.
     */
    @ConfigItem
    public boolean clustered;
 ```